samples/sdl_gamecontroller/main.c

```diff
@@ -0,0 +1,105 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/video.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
```

Comment:
nit: It's less of an issue here since the whole sample is formatted equally, but I still recommend to stick to 4 spaces like most of nxdk.

(we should come up with a way to avoid having to deal with indentation, line-endings etc. manually - [EditorConfig](https://editorconfig.org/) maybe?)

Reply:
Please create an issue about this, if none exists yet @thrimbor ; then mark as resolved.

Reply:
Resolving as unrelated. No issue has been created yet.